packages/create-ripple/src/lib/project-creator.js

```diff
@@ -164,6 +177,21 @@ function updatePackageJson(projectPath, projectName, packageManager, typescript)
 		packageJson.packageManager = getPackageManagerVersion(packageManager);
 	}
 
+	// Add styling dependencies
+    if (stylingFramework === 'tailwind') {
+        packageJson.devDependencies = {
+            ...packageJson.devDependencies,
+            'tailwindcss': '^3.0.0',
```

Comment:
Use tailwind v4 (we don't want to have old dependencies - tailwind v4 have also fewer dependencies).